templates/SimpleInventory/edit-board.html

```diff
@@ -0,0 +1,45 @@
+{% extends "SimpleBase/base.html" %}
+
+{% block head %}
+{{ block.super }}
+{% load static %}
+<link rel="stylesheet" href="{% static 'SimpleInventory/css/orderboard.css' %}">
+{% endblock %}
+
+{% block js %}
+<div id="message-toast" class="mdl-js-snackbar mdl-snackbar">
+  <div class="mdl-snackbar__text"></div>
+  <button class="mdl-snackbar__action" type="button"></button>
+</div>
+{{ block.super }}
+{% load static %}
+<script src="{% static 'SimpleInventory/js/edit-board.js' %}"></script>
+{% endblock %}
+
+{% block content %}
+<div id="formCard" class="form-card mdl-card mdl-cell--8-col mdl-cell--2-offset mdl-shadow--2dp">
+    <div class="mdl-card__title">
+        <h2 class="mdl-card__title-text">Edit Board</h2>
+    </div>
+    <div class="mdl-card__supporting-text">
+        <form method="post" action="{% url 'api_v1_board_update' board_id %}" id="editBoardForm" class="mdl-cell--2-offset mdl-cell--8-col">
+            {% csrf_token %}
+            {{ form.active.as_hidden }}
+            <div class="mdl-textfield mdl-js-textfield mdl-textfield--floating-label mdl-cell--12-col mdl-cell--5-col-tablet mdl-cell--bottom">
```

Comment:
Having these forms shink to 5 and 3 columns on tablet doesn't work out here.  Again, this is an issue I introduced in the form you copied this from, so I can fix it on those other forms.  For here, you should just be able to delete the tablet-specific classes and make sure it looks good, both on this and description div.